### PR TITLE
Assuming that if a dot is present in a table name, that the right

### PR DESCRIPTION
**Pull request by patrickconroy**
…side is the real name.

(I'll write a test for this if people agree that this is the right thing to do.)

References #6539

### CODE REVIEW

File: src/Core/ConventionsTrait.php [hunk 1]
```diff
@@ -48,12 +48,17 @@ protected function _entityName($name)
...
     /**
      * Creates the proper underscored model key for associations
+     * If the input contains a dot, assume that the right side is the real table name.
      *
      * @param string $name Model class name
      * @return string Singular model key
      */
     protected function _modelKey($name)
     {
+        $model = explode('.', $name);
```
Comment (ADmad) [inline src/Core/ConventionsTrait.php]: Can just do `list(, $name) = pluginSplit($name);`

Comment (markstory) [inline src/Core/ConventionsTrait.php]: :+1:

Comment (patrickconroy) [inline src/Core/ConventionsTrait.php]: Yeah that makes way more sense, just pushed that.

**Comment by markstory** [comment]
It might be good to have a test for this in the table/association tests so we don't accidentally regress in the future.

File: src/Core/ConventionsTrait.php [hunk 2]
```diff
@@ -48,12 +48,14 @@ protected function _entityName($name)
 
     /**
      * Creates the proper underscored model key for associations
+     * If the input contains a dot, assume that the right side is the real table name.
```
Comment (ravage84) [inline src/Core/ConventionsTrait.php]: A blank doc block line above, please.

**Comment by patrickconroy** [comment]
Thanks!

**Pull request merged**